utils/fetch.js

```diff
@@ -44,18 +44,15 @@ export function populateChain(chain, chainTvls) {
 
     const chainSlug = chainIds[chain.chainId];
 
-    if (chainSlug !== undefined) {
-        const defiChain = chainTvls.find((c) => c.name.toLowerCase() === chainSlug);
```

Comment:
Note: Changed tvl matching and using chainId for matching because chain name does not match in all cases like rootstock